src/react/VGrid.tsx

```diff
@@ -149,6 +151,12 @@ export interface VGridHandle {
    * @param indexY vertical of item
    */
   getItemSize(indexX: number, indexY: number): [width: number, height: number];
+  /**
+   * Resize individual columns and rows.
+   * @param cols array of `[index, size]` to update column sizes
+   * @param rows array of `[index, size]` to update row sizes
+   */
+  resize(cols: VGridItemResize[], rows: VGridItemResize[]): void;
```

Comment:
I felt it might be better to separate API for cols and rows. The current API is unclear what happens if both cols and rows are passed, and also that would be a rare usecase.

```suggestion
  resizeCols(cols: VGridItemResize[]): void;
  resizeRows(rows: VGridItemResize[]): void;
```